predicates/tee/tee.go

```diff
@@ -0,0 +1,37 @@
+package tee
+
+import (
+	"github.com/zalando/skipper/predicates"
+	"github.com/zalando/skipper/routing"
+	"net/http"
+)
+
+const (
+	// The eskip name of the predicate.
+	PredicateName = "Tee"
+	HeaderKey     = "x-tee-loopback-key"
+)
+
+type spec struct{}
+
+type predicate struct {
+	key string
+}
+
+func New() routing.PredicateSpec { return &spec{} }
+
+func (s *spec) Name() string { return PredicateName }
+
+func (s *spec) Create(args []interface{}) (routing.Predicate, error) {
+	if len(args) != 1 {
+		return nil, predicates.ErrInvalidPredicateParameters
+	}
+	return &predicate{
+		key: args[0].(string),
```

Comment:
please check if it's a string, otherwise it may panic. Similar to the teeLoopback filter args.